ui/src/components/Region/Colocalization/ColocalizationContext.tsx

```diff
@@ -23,18 +24,41 @@ export interface ColocalizationState {
 export const ColocalizationContext = createContext<Partial<ColocalizationState>>({});
 
 
+export interface RegionMax {
+    readonly region : Region;}
+
 const ColocalizationContextProvider = ({ params , children} :  Props) => {
+
     const [colocalization, setColocalization] = useState<Colocalization[]| undefined>(undefined);
     const [locusZoomData, setLocusZoomData] = useState<LocusZoomData| undefined>(undefined);
     const [selectedColocalization, setSelectedColocalization] = useState<Colocalization | undefined>(undefined);
     const [searchSummary, setSearchSummary] = useState<SearchSummary | undefined>(undefined);
     const [casualVariant, selectedCasualVariant] = useState<CasualVariant | undefined>(undefined);
-        useEffect(() => {
+    const [maxRegion, setMaxRegion] = useState< RegionParams<Locus>| undefined>(undefined);
+    const [region, setRegion] = useState<Region| undefined>(undefined); 
+    
+    useEffect(() => {
+        const parameter : RegionParams<Locus>| undefined = createParameter(params)
+        getRegion(parameter,setRegion); }, [params]);
+    
+    useEffect (() => {
+        const locus = {
+            'chromosome': region?.cond_fm_regions[1]['chr'],
+            'start': region?.cond_fm_regions[1]['start'],
+            'stop': region?.cond_fm_regions[1]['end']
+        } 
+        setMaxRegion({'locus': locus, 'phenotype': params?.phenotype})
```

Comment:
here I believe cond_fm_regions contain both finemap and conditional regions. 
They can be potentially different so should set the coloc region to region of finemapped specifically

Reply:
right, indeed instead of accessing by index will do some more precise search for the finemapped